### DISCUSSION TITLE
LiDAR Sensor Data

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#106] [General]
# LiDAR Sensor Data

Coordinate Frames and XYZ Calculation Lidar Coordinate Frame 은 오른손 법칙을 따르며, point clouds 를 계산하고 조작할 수 있는 가장 간단한 frame 의 point cloud-centric frame 이다. X 좌표는 외부 커넥터를 향해 뒤쪽을 향하고 있는데, 이는 직관적이지 않은 방향이며 다음…

https://vero.wiki/lidar-sensor-data/

**Comment by ukkodeveloper** [comment]
와 멋있다 이게 전공자구나

**Reply by Cyma-s** [reply]
나도 아직 잘 몰라 ㅋㅋㅋㅋㅋ
